benchmarks/perf-tool/sample-configs/faiss-nsg/test.yml

```diff
@@ -0,0 +1,36 @@
+endpoint: localhost
+test_name: faiss_nsg
+test_id: "Test workflow for faiss nsg"
+num_runs: 2
+show_runs: true
+setup:
+  - name: delete_index
+    index_name: target_index
+steps:
+  - name: create_index
+    index_name: target_index
+    index_spec: sample-configs/faiss-nsg/index-spec.json
+  - name: ingest
+    index_name: target_index
+    field_name: target_field
+    bulk_size: 500
+    dataset_format: hdf5
+    dataset_path: ../dataset/sift-128-euclidean.hdf5
+  - name: refresh_index
+    index_name: target_index
+  - name: force_merge
+    index_name: target_index
+    max_num_segments: 10
```

Comment:
same as above.

Reply:
in sample-config, i copied from https://github.com/opensearch-project/k-NN/blob/5b26be318e71fed55efb7f1038bbfcb44fc06029/benchmarks/perf-tool/sample-configs/nmslib-sift-hnsw/test.yml#L21-L23

i just want to diff the performance with the same shards